dataflow/run-inference/requirements-test.txt

```
@@ -1,4 +1,4 @@
-google-cloud-aiplatform==1.57.0
+google-cloud-aiplatform==1.90.0
```

```suggestion
google-cloud-aiplatform==1.57.0
```

revert update for package failing for unrelated reasons